app/Http/Controllers/UserController.php

```diff
@@ -421,19 +440,61 @@ public function extendWorkmail()
         }
     }
 
+    /**
+     * Fetch users from VATSIM Core API
+     *
+     * @return \Illuminate\Http\Response|bool
+     */
+    private function fetchUsersFromVatsimCoreApi()
+    {
+        $url = sprintf('https://api.vatsim.net/v2/orgs/subdivision/%s', config('app.owner_short'));
+        $headers = [
+            'X-API-Key' => config('vatsim.core_api_token'),
+            'Accept' => 'application/json',
+            'Content-Type' => 'application/json',
+        ];
+
+        $users = [];
+        $usersCount = 0;
+
+        $limit = 1000;
```

Comment:
I am not sure how big should `limit` be. We have only 172 members, so only 1 request has to be made. But I can imagine that there are divisions with lot of members, so as precaution, I implemented do-while loop to cover such case. Api default is 100, which is quite low imho.

Reply:
Nah, I mean I don't see a reason why we need that as a configurable option. If it's above the total members of your subdivision it'll just return everything anyway. I'd say it's more useful for pagination, which is not relevant here.